autoload/wiki/fzf.vim

```diff
@@ -17,11 +17,14 @@ function! wiki#fzf#pages() abort "{{{1
   let l:pages = globpath(l:root, l:pattern, v:false, v:true)
   call map(l:pages, '"/" . substitute(v:val, l:root . "/" , "", "")')
   call map(l:pages, {_, x -> x . "¤" . fnamemodify(x, ':r')})
-
+  let fzf_opts = join([
```

Comment:
I prefer to keep the empty line above to separate blocks of code. Also, although it is clearly opinionated, I've settled on the style of prefixing the `l:` to indicate local variables.

Reply:
Sure thing, I'll include these style changes in the next revision.